cookbooks/aws-parallelcluster-config/files/default/cloudwatch_agent/write_cloudwatch_agent_json.py

```diff
@@ -159,10 +202,20 @@ def create_config(log_configs):
     }
 
 
+def get_dict_value(value, attributes, default=None):
```

Comment:
nit: probably this is probably getting an attribute from nested classes rather than a dict.